app/controllers/search_controller.rb

```diff
@@ -0,0 +1,5 @@
+class SearchController < ApplicationController
+  def create
+    @results = Question.search_for(params[:search])
```

Comment:
This should be able to search for Questions and Answers.  Checkout pg_search multisearch.